src/common/packager.ts

```diff
@@ -393,4 +402,8 @@ export class Packager {
 
         return atomScript;
     }
+
+    private getSourceExtensions(): Array<string> {
```

Comment:
Please add a commentary with an explanation of why we need that function with a link on the original function.